build.gradle.kts

```diff
@@ -19,13 +18,6 @@ plugins {
 }
 
 allprojects {
-    plugins.withType<KotlinBasePlugin>().configureEach {
-        dependencies {
-            // https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library
-            "compileOnly"(libs.kotlin.stdlib)
-        }
-    }
-
```

Comment:
I believe we can keep this here.

Reply:
I intentionally decided to add this explicitly to the necessary modules because it became harder to maintain the code.